src/components/Layout.tsx

```diff
@@ -88,23 +88,27 @@ function Header() {
 export function Layout({ children }: { children: React.ReactNode }) {
 	let pathname = usePathname();
 	let isHomePage = pathname === '/';
+	let isCodyApiPage = pathname === '/cody-api';
```

Comment:
The OpenAPI spec is for "Sourcegraph API" and I think it's best that we update the page https://sourcegraph.com/docs/api to be the home for these docs. I will update the OpenAPI spec to include the contents from the current https://sourcegraph.com/docs/api/stream_api https://sourcegraph.com/docs/api/graphql pages

The end goal is that these API docs will cover all of our APIs, not just Cody.

Reply:
After more thinking, it's probably best to call this the "OpenAPI explorer" for "Sourcegraph APIs". We can treat this as a sibling page to the graphql and streaming search. The OpenAPI explorer will also document streaming search + graphql, but it's OK if there's some duplicate information. We can't remove the custom streaming search page or graphql page